filebeat/module/logstash/slowlog/ingest/pipeline-json.json

```diff
@@ -0,0 +1,87 @@
+{
+    "description": "Pipeline for parsing logstash log logs",
+    "on_failure": [
+        {
+            "set": {
+                "field": "error.message",
+                "value": "{{ _ingest.on_failure_message }}"
+            }
+        }
+    ],
+    "processors": [
+        {
+            "json": {
```

Comment:
The JSON decoding could also be done on the Filebeat side, but I'm not sure if there's any advantage to that.